src/repository/opamDownload.ml

```diff
@@ -246,8 +246,13 @@ module SWHID = struct
   let check_liveness () =
     OpamProcess.Job.catch (fun _ -> Done false)
     @@ fun () ->
-      get_output ~post:true OpamUrl.Op.(instance / "api" / "1" / "ping" / "")
-      @@| fun _ -> true
+    get_output ~post:false OpamUrl.Op.(instance / "api" / "1" / "ping" / "")
+    @@| function
+    | Some (pong::_) ->
+      (* curl output after answer the http code *)
```

Comment:
I don't follow the comment - are there situations where curl is doing something different with the output?

Reply:
curl invocation contains `--write-out "%{http_code}\n"`, so its output on stdout is `"pong"200`

Reply:
Oh, I was being silly - I didn't register that `get_output`'s `cmd_args` of course begins with a call to `download_args` 🤦‍♂️